src/cmd-buildupload

```diff
@@ -119,6 +132,13 @@ def s3_upload_build(args, builddir, dest):
               '--content-type=application/json')
 
 
+def s3_check_exists(args, path):
```

Comment:
`S3Fetcher` already has   `def exists_impl(self, url):`

Reply:
`S3Fetcher` lives inside of `cmd-buildprep` at the moment which cannot be imported because modules can't have `-` in the name. Do you want me to move it into `cmdlib` as well?

Reply:
Oh yeah.  Eh.   It's OK for now.